### PR TITLE
bz1944506: added alternative actions.

### PR DESCRIPTION
**Pull request by nalhadef**
For 4.6+

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1944506

Direct link to doc preview: https://deploy-preview-36303--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-tailor?utm_source=github&utm_campaign=bot_dp#compliance-tailored-profiles_compliance-tailor

Ready for QA: @pdhamdhe 
Ready for SME: @JAORMX

### CODE REVIEW

File: modules/compliance-tailored-profiles.adoc [hunk 1]
```diff
@@ -46,7 +46,32 @@ spec:
...
+
+|`disableRules`
+|A list of name and rationale pairs. Each name refers to a name of a rule object that is to be disabled. The rationale value is human-readable text describing why the rule is disabled.
+
+|`enableRules`
+|A list of name and rationale pairs. Each name refers to a name of a rule object that is to be enabled. The rationale value is human-readable text describing why the rule is enabled.
+
+|`description`
+|Human-readable text describing the `TailoredProfile`.
+|===
```
Comment (pdhamdhe) [inline modules/compliance-tailored-profiles.adoc:73]: Can we have `setValues`attributes description in above table ?

Comment (nalhadef) [reply modules/compliance-tailored-profiles.adoc:76]: Thanks! I've added setValues.

File: modules/nw-exposing-service.adoc [hunk 2]
```diff
@@ -85,7 +85,7 @@ ifdef::nodeport[]
 +
 [source,terminal]
 ----
-$ oc expose dc mysql-80-rhel7 --type=NodePort --name=mysql-ingress
+$ oc expose svc/mysql-80-rhel7 --type=NodePort --name=mysql-nodeport --generator="service/v2"
 ----
```
Comment (pdhamdhe) [inline modules/nw-exposing-service.adoc:89]: @nalhadef Just to confirm, Is this part of our PR ?

Comment (nalhadef) [reply modules/nw-exposing-service.adoc]: Thanks, @pdhamdhe. I've removed the erroneous entries.

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**